rust-maven-example/src/test/java/io/questdb/example/rust/LibTest.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.example.rust;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+
+import org.junit.Test;
+
+import io.questdb.jar.jni.LibInfo;
+
+public class LibTest {
+
+    @Test
+    public void testLibrary() {
+        File libraryFile = new File("target/rust-maven-plugin/str-reverse/release/", new LibInfo("str_reverse").getFullName());
+
+        System.load(libraryFile.getAbsolutePath());
```

Comment:
This shouldn't be necessary once the loading code is moved to a `static` block.

Did you encounter issues?

Reply:
It might have been a fluke of my IDE. I thought Class.getResourceAsStream wasn't working in the test, but checking it again, it seems to work fine.